### PR TITLE
Added color pallete

### LINKED ISSUE #573: Add color picker in `Filter Images` section
screenshot 2019-02-26 at 11 02 50 pm

- select **Images to PDF**
- Select few images. select filter images option
- select brush icon

Currently, we can select only a limited number of colors for brush. 

### What to do?
- At the end of the color boxes that show up on selecting the brush, add an icon. 
- on selecting that icon, open up a color picker dialog (We already are using a color picker library in watermark section. So, use that only)

### PR DESCRIPTION
**Pull request by jddeep**
# Description
Added the color pallete while filtering the image.

screenshot_20190302-131409

Fixes #573 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by jddeep** [comment]
@Swati4star please review

**Comment by jddeep** [comment]
@Swati4star and why is the ci build failing. The build is fine on my local machine. Here i can see some AAPT error regarding linux. Can you provide some help?

**Comment by Swati4star** [comment]
@jddeep Why are you showing the exact same dialog as watermark? 
Use the same color picker. The watermark options don'tseem valid here.

File: app/src/main/res/layout/activity_photo_editor.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
             android:background="@drawable/cornered_edges"
             android:paddingStart="5dp"
             android:paddingEnd="5dp"
-            android:text="@string/save_current"
+            android:text="@string/save"
```
Comment (Swati4star) [inline app/src/main/res/layout/activity_photo_editor.xml]: There is not string named "save" in resources. That's why the build is failing

File: app/src/main/java/swati4star/createpdf/activity/ImageEditor.java [hunk 2]
```diff
@@ -81,7 +85,8 @@ protected void onCreate(Bundle savedInstanceState) {
         ButterKnife.bind(this);
 
         initValues();
-
+        mWatermarkutils = new WatermarkUtils(this);
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/activity/ImageEditor.java]: There should be no use of watermark here.

File: app/src/main/res/layout/activity_photo_editor.xml (hunk 1 above)
Comment (jddeep) [reply app/src/main/res/layout/activity_photo_editor.xml]: Oh i didn't commit the strings.xml file

File: app/src/main/java/swati4star/createpdf/activity/ImageEditor.java (hunk 2 above)
Comment (jddeep) [reply app/src/main/java/swati4star/createpdf/activity/ImageEditor.java]: Okay

**Comment by jddeep** [comment]
@Swati4star Done. Please review now. Thanks.

File: app/src/main/java/swati4star/createpdf/activity/ImageEditor.java (hunk 2 above)
Comment (Swati4star) [reply app/src/main/java/swati4star/createpdf/activity/ImageEditor.java]: We dont need watermarkutils now. Remove it & corresponding imports too

File: app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java [hunk 3]
```diff
@@ -344,6 +343,15 @@ public void onAttach(Context context) {
         mDirectoryUtils = new DirectoryUtils(mActivity);
     }
 
+    @Override
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java]: this change shouldnt be here

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -188,6 +188,8 @@
     <string name="decrypt_message">Enter the password to continue</string>
     <string name="not_encrypted">PDF is not Encrypted</string>
     <string name="save_current">Save Current</string>
+    <string name="save">Save</string>
+    <string name="choose_color_text">Choose a color :</string>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml]: Add these strings for other locale files also (no need to translate - add them in english only)

**Comment by Swati4star** [comment]
Your commit is messed up. You havent committed BrushItemAdapter file, and now app is crashing.

**Comment by jddeep** [comment]
I commited particular files. Now i guess i need to commit all to avoid such issues in the future. Will update the commit.

**Comment by Swati4star** [comment]
This is the reason, I asked you to create a different branch for new issue

**Comment by jddeep** [comment]
Yeah that's why i even created.

**Comment by jddeep** [comment]
@Swati4star Please review.

**Pull request merged**